centaur/src/main/resources/standardTestCases/read_tsv.test

```diff
@@ -1,5 +1,8 @@
 name: read_tsv
 testFormat: workflowsuccess
+backendsMode: "only"
+backends: [Local, LocalNoDocker]
+tags: [localdockertest]
```

Comment:
I think this warrants a ticket so that this sort of WDL feature works on PAPI as we discussed with @cjllanwarne last week.